platforms/documentation/docs/src/docs/userguide/reference/runtime-configuration/gradle_daemon.adoc

```diff
@@ -16,6 +16,39 @@
 = Gradle Daemon
 :keywords: daemon, status, jps, no-daemon, stop, org.gradle.daemon, updateDaemonJvm, gradle-daemon-jvm.properties, jvm-version, jvm-vendor
 
+The Gradle Daemon is a long-lived, persistent process that runs in the background and hosts Gradle's execution engine.
+It dramatically reduces build times using caching, runtime optimizations, and eliminating JVM startup overhead.
+
+== The Gradle Client vs. the Gradle Daemon
+
+To run any Gradle command, your system starts two separate Java Virtual Machine (JVM) processes, which can use different Java versions.
+
+Understanding their distinct roles and how their respective JVM versions are determined is key to effective build configuration and debugging:
+
+[cols="1,2,3", options="header"]
+|===
+| Process | Role | JVM Version Source
+
+| *Gradle Client JVM*
+| Process that starts when you run `./gradlew` and stays alive for the *entire build invocation*. +
+It connects to (or starts) a Daemon, sends the build request, and streams output back to the console.
+| Uses the JVM that launches the wrapper script (e.g. `JAVA_HOME`, `java` on `PATH`, or your IDE).
+
+| *Gradle Daemon JVM*
+| Long-lived process that executes the build and caches state between runs.
+| Uses the JVM set by (in order): +
+1) `--java-home` (CLI) +
+2) `org.gradle.java.home` (properties) +
+3) <<tooling_api.adoc#tooling_api,Tooling API>> requests (e.g., IDE) +
+4) <<#sec:configuring_daemon_jvm,Daemon JVM toolchains>> +
+5) Falls back to the JVM that launched the Gradle Client or reuses an existing Daemon if conditions are right*
```

Comment:
I think this is conflating two different things.

1. Given no other opinions, Gradle will use the JVM from the client.  
2. The client will look for compatible daemons and, by default, start a new one if none are available.

We only attempt to use the client process to run the build if the daemon has been disabled (--no-daemon or -Dorg.gradle.daemon=false) and the client process is compatible. It almost never is compatible, so the client will start a single use daemon instead.

Reply:
I also re-wrote this more times than I can count. Let me know what you think of this newer instance. I aimed to make this as simple as possible (but I oversimplified imho).